src/components/reporter/widgets/TextArea.js

```diff
@@ -12,25 +12,46 @@ const TextArea = ({ name, required, fieldState, isInvalid, onChange }) => {
     onChange(e);
   }
 
+  const checkPageLocation = () => {
+    if (typeof window !== `undefined`) {
+      return window.location.href.includes("/help");
+    }
+  };
```

Comment:
Instead of adding this specific logic into the TextArea component (the TextArea should not have to be aware of the location) let's add an optional prop to this component for `maxSize` and pass that from the `help` consumer.

Reply:
I see, I'll adapt accordingly